.gitignore

```diff
@@ -142,7 +142,6 @@ node_modules/
 *.Tests/lib/
 *.userprefs
 *.vsp
-*.xml
```

Comment:
Not sure why all `.xml` files are excluded; perhaps we should exclude a specific location instead? This value makes it harder to add .xml files (such as `SourceBuildPrebuiltBaseline.xml`) to the sources.

Reply:
We don't need to exclude `.xml`.